app/Utils/dotpathUtil.php

```diff
@@ -0,0 +1,207 @@
+				$rssItem['content'] = isset($dotPaths['itemContentHTML'])
+					? FreshRSS_dotpath_Util::getString($jsonItem, $dotPaths['itemContentHTML']) ?? ''
+					: $rssItem['content'];
+
+				if (isset($dotPaths['itemTimeFormat']) && is_string($dotPaths['itemTimeFormat'])) {
+					$dateTime = DateTime::createFromFormat($dotPaths['itemTimeFormat'], $rssItem['timestamp']);
+					if ($dateTime != false) {
+						$rssItem['timestamp'] = $dateTime->format(DateTime::ATOM);
+					}
+				}
+
+				if (isset($dotPaths['itemCategories'])) {
+					$jsonItemCategories = FreshRSS_dotpath_Util::get($jsonItem, $dotPaths['itemCategories']);
+					if (is_string($jsonItemCategories) && $jsonItemCategories !== '') {
+						$rssItem['tags'] = [$jsonItemCategories];
+					} elseif (is_array($jsonItemCategories) && count($jsonItemCategories) > 0) {
+						$rssItem['tags'] = [];
+						foreach ($jsonItemCategories as $jsonItemCategory) {
+							if (is_string($jsonItemCategory)) {
+								$rssItem['tags'][] = $jsonItemCategory;
+							}
+						}
+					}
+				}
+
+				$rssItem['thumbnail'] = isset($dotPaths['itemThumbnail']) ? FreshRSS_dotpath_Util::getString($jsonItem, $dotPaths['itemThumbnail']) ?? '' : '';
+
+				//Enclosures?
+				if (isset($dotPaths['itemAttachment'])) {
+					$jsonItemAttachments = FreshRSS_dotpath_Util::get($jsonItem, $dotPaths['itemAttachment']);
+					if (is_array($jsonItemAttachments) && count($jsonItemAttachments) > 0) {
+						$rssItem['attachments'] = [];
+						foreach ($jsonItemAttachments as $attachment) {
+							$rssAttachment = [];
+							$rssAttachment['url'] = isset($dotPaths['itemAttachmentUrl'])
+								? FreshRSS_dotpath_Util::getString($attachment, $dotPaths['itemAttachmentUrl'])
+								: '';
+							$rssAttachment['type'] = isset($dotPaths['itemAttachmentType'])
+								? FreshRSS_dotpath_Util::getString($attachment, $dotPaths['itemAttachmentType'])
+								: '';
+							$rssAttachment['length'] = isset($dotPaths['itemAttachmentLength'])
+								? FreshRSS_dotpath_Util::get($attachment, $dotPaths['itemAttachmentLength'])
+								: '';
+							$rssItem['attachments'][] = $rssAttachment;
+						}
+					}
+				}
+
+				if (isset($dotPaths['itemUid'])) {
+					$rssItem['guid'] = FreshRSS_dotpath_Util::getString($jsonItem, $dotPaths['itemUid']);
+				}
+
+				if (empty($rssItem['guid'])) {
+					$rssItem['guid'] = 'urn:sha1:' . sha1($rssItem['title'] . $rssItem['content'] . $rssItem['link']);
+				}
+
+				if ($rssItem['title'] != '' || $rssItem['content'] != '' || $rssItem['link'] != '') {
+					// HTML-encoding/escaping of the relevant fields (all except 'content')
+					foreach (['author', 'guid', 'link', 'thumbnail', 'timestamp', 'tags', 'title'] as $key) {
+						if (!empty($rssItem[$key]) && is_string($rssItem[$key])) {
+							$rssItem[$key] = Minz_Helper::htmlspecialchars_utf8($rssItem[$key]);
+						}
+					}
+					$view->entries[] = FreshRSS_Entry::fromArray($rssItem);
+				}
+			}
+		} catch (Exception $ex) {
```

Comment:
It does not look there was anything to catch there.
Addressed in https://github.com/FreshRSS/FreshRSS/pull/6037